py_neuromodulation/nm_filter.py

```diff
@@ -1,13 +1,12 @@
 """Module for filter functionality."""
 
-import logging
-
-logger = logging.getLogger("PynmLogger")
-
 import mne
 from mne.filter import _overlap_add_filter
 import numpy as np
+from typing import cast
 
+import logging
+logger = logging.getLogger("PynmLogger")
```

Comment:
I guess those are the same changes as in the logger PR?

Reply:
Not exactly, this was just moving the logger initialization below all the other imports, so that linters would not complain about imports not being on top xD Just an aesthetic thing